packages/react/src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -2,13 +2,19 @@ import { pxToRem } from '../../../../lib'
 
 export interface MenuVariables {
   color: string
+  borderColor: string
+  verticalBorderColor: string
 
-  activeColor: string
-  activeBackgroundColor: string
+  focusedBorder: string
```

Comment:
These are breaking changes. Please in the description of the PR explain how the usages of the removed variables should be handle.